cppcache/src/statistics/AtomicStatisticsImpl.cpp

```diff
@@ -263,12 +261,12 @@ int64_t AtomicStatisticsImpl::_incLong(int32_t offset, int64_t delta) {
      * callers must be careful not to overflow the actual space of the
      * destination.
      * Use snprintf() instead, or correct precision specifiers.
-     * Fix : using ACE_OS::snprintf
+     * Fix : using std::snprintf
      */
     // sprintf(s, "incLong:The id (%d) of the Statistic Descriptor is not valid
```

Comment:
Delete commented-out code

Reply:
This was part of a global search and replace. I generally preserve cleanup of surrounding code for areas I am actually working in.